src/ensureArray.js

```diff
@@ -0,0 +1,23 @@
+import { when, of } from 'ramda';
+import { isNotArray } from './index';
+
+/**
+ * If the argument is already an Array, return it,
```

Comment:
I'd prefer the following as description. It's basically a compilate of `when` and `of` descriptions.
```
Returns a singleton array containing the value provided.
If value is already an array, it is returned as is.